src/aks-preview/azext_aks_preview/_validators.py

```diff
@@ -309,6 +313,14 @@ def validate_load_balancer_idle_timeout(namespace):
                 "--load-balancer-idle-timeout must be in the range [4,100]")
 
 
+def validate_load_balancer_backend_pool_type(namespace):
+    """validate load balancer backend pool type"""
+    if namespace.load_balancer_backend_pool_type is not None:
+        if namespace.load_balancer_backend_pool_type not in [CONST_LOAD_BALANCER_BACKEND_POOL_TYPE_NODE_IP, CONST_LOAD_BALANCER_BACKEND_POOL_TYPE_NODE_IPCONFIGURATION]:
+            raise CLIError(
```

Comment:
More detailed error type would be better, I would suggest `InvalidArgumentValueError`.

Reply:
done